### DISCUSSION TITLE
在深色模式下建议将theme-color设为黑色

### DISCUSSION THREAD

**Discussion by 1llilhz** [the1812/Bilibili-Evolved#2384] [功能建议 / Ideas]
Safari 15更新了匹配主题颜色，但深色模式下还是显示原网页主题色（蓝色）

**Comment by the1812** [comment]
这是 Safari 的特殊功能吗, 我应该在哪设置

**Comment by 1llilhz** [comment]
你好， 是Safari的新增功能，具体实现就是将上方红框标签页蓝的颜色转为网页中的Theme-color，具体的设置是在下方meta标签。




> 2021年10月12日 21:08，Grant Howard ***@***.***> 写道：
> 
> 
> 这是 Safari 的特殊功能吗, 我应该在哪设置
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/the1812/Bilibili-Evolved/discussions/2384#discussioncomment-1464698>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJE2WSJAML4RTLZJXH6FXITUGQXMTANCNFSM5FV74UXA>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> 

<img width="1792" alt="截屏2021-10-13 13 38 04" src="https://user-images.githubusercontent.com/38382409/137074095-840e0c86-4b49-4a5f-a807-f36463c2f339.png">
